backend/src/resolvers/users.spec.js

```diff
@@ -5,15 +5,15 @@ import Factory from '../seed/factories'
 const factory = Factory()
 let client
 
-afterAll(async () => {
+afterEach(async () => {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-10T15:39:59Z" title="Friday, May 10th 2019, 5:39:59 pm +02:00">May 10, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -5,15 +5,15 @@ import Factory from '../seed/factories'
 const factory = Factory()
 let client
 
-afterAll(async () => {
+afterEach(async () => {
```


@ulfgebhardt `afterEach` means: Run this after each test case. `afterAll` means: Run this after the entire test **suite**. `afterAll` is great for performance tweaks, but you have to tear down the created data yourself. Read: https://jestjs.io/docs/en/setup-teardown